src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -56,10 +56,21 @@ internal class AuthenticationRequestParameters
             ClientId = serviceBundle.Config.ClientId;
             TokenCache = tokenCache;
             Scope = ScopeHelper.CreateSortedSetFromEnumerable(commonParameters.Scopes);
-            ExtraQueryParameters = commonParameters.ExtraQueryParameters ?? new Dictionary<string, string>();
             RedirectUri = new Uri(serviceBundle.Config.RedirectUri);
             RequestContext = requestContext;
             ApiId = commonParameters.ApiId;
+
+            // Set application wide query parameters.
+            ExtraQueryParameters = serviceBundle.Config.ExtraQueryParameters ?? new Dictionary<string, string>();
+
+            // Copy in call-specific query parameters.
+            if (commonParameters.ExtraQueryParameters != null)
```

Comment:
what appens if the parameter already exists? do I understand that it's overridden in the AuthenticationRequestParameter ?